rasa/shared/nlu/training_data/util.py

```diff
@@ -103,6 +104,16 @@ def remove_untrainable_entities_from(example: Dict[Text, Any]) -> None:
     example[ENTITIES] = trainable_entities
 
 
+def intent_response_key_to_template_key(intent_response_key: Text) -> Text:
+    """Resolve the response template key for a given intent response key."""
+    return f"{UTTER_PREFIX}{intent_response_key}"
+
+
+def template_key_to_intent_response_key(template_key: Text) -> Text:
+    """Resolve the intent response key for a given response template key."""
+    return template_key.split(UTTER_PREFIX)[1]
+
+
```

Comment:
Please use full docstring which also describe the parameters + return values.

Reply:
I skipped here because the function names and arguments were very descriptive. I'll still add them in.

Reply:
Makes sense but we need to strive for consistency.